### PR TITLE
[IMP] accounting: mexico - MXN as default currency

### PR DESCRIPTION
**Pull request by adwid**
A Mexican company should keep MXN as default company. This is actually a
legal obligation.

### CODE REVIEW

**Review by nle-odoo — COMMENTED** [review]
Hi @jcs-odoo 


I'm note sure of who has merge power or usually review use documentation change. 

So could you check this or redirect us to the right person.

Thanks )

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello!

PR can be merged once approved by another user.
So it's good you pinged me :)

I just need you to do two small changes to your commit before we merge it:
- the indentation of the `warning` message is wrong. See the suggestion
- could you rename the commit like this, please?
  `[IMP] accounting: add a warning about currency in Mexican localization`

Thanks

File: accounting/localizations/mexico.rst [hunk 1]
```diff
@@ -81,6 +81,12 @@ company’s contact.
 .. image:: media/mexico03.png
    :align: center
 
+.. warning::
+    From a legal point of view, a Mexican company must use the local currency (MXN).
+    Therefore, Odoo does not provide features to manage an alternative configuration.
+    If you want to manage another currency, let MXN be the default currency and use
+    price list instead.
+
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply accounting/localizations/mexico.rst:89]: oh and for content changes or addition, please add the label `content improvement` and also add the version on which the commit is based as a `milestone`. It's already done for this pr :)

**Comment by adwid** [comment]
@jcs-odoo Thank for the review!
Now it should be ok

**Review by jcs-odoo — APPROVED** [review]
All good 👍

**Pull request merged**